handbook/openid-connect/openid-connect.md

```diff
@@ -316,3 +316,18 @@ The following sources can be used to troubleshoot a failed connection.
     SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.
 
 **Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).
+
+### Cache Shm Entry Size Max
+
+If an "Internal Server Error" is encountered when attempting to log in, this may be an issue with the defined `OIDCCacheShmEntrySizeMax`. You can verify this is the issue by searching the NI Web Server logs for the following entries
```

Comment:
Might be good to have a little higher-level description here, something along the lines of:
```suggestion
The OpenID Connect module stores information in a shared memory cache. If the cache entry is too small, users will see an internal server error when attempting to log in, and the NI Web Server logs will contain the following entries:
```